core/shared/src/test/scala/fs2/HashSuite.scala

```diff
@@ -45,30 +42,30 @@ class HashSuite extends Fs2Suite {
             acc ++ Stream.chunk(Chunk.array(c))
           )
 
-    assertEquals(s.through(h).toList, digest(algo, str))
+    s.through(h).compile.toList.assertEquals(digest(algo, str))
   }
 
   group("digests") {
-    test("md2")(forAll((s: String) => checkDigest(md2, "MD2", s)))
-    test("md5")(forAll((s: String) => checkDigest(md5, "MD5", s)))
-    test("sha1")(forAll((s: String) => checkDigest(sha1, "SHA-1", s)))
-    test("sha256")(forAll((s: String) => checkDigest(sha256, "SHA-256", s)))
-    test("sha384")(forAll((s: String) => checkDigest(sha384, "SHA-384", s)))
-    test("sha512")(forAll((s: String) => checkDigest(sha512, "SHA-512", s)))
+    if (isJVM) test("md2")(forAllF((s: String) => checkDigest(md2, "MD2", s)))
```

Comment:
So, crypto is weird, since ultimately it depends on availability of algorithms at runtime. Seems that default OpenSSL builds (also used by Node.js) don't include MD2. But an alternative build could.